kubetest/kind/kind.go

```diff
@@ -60,9 +61,10 @@ var (
 		"(kind only) name:tag of the base image to use for building the node image for kind.")
 	kindBinaryVersion = flag.String("kind-binary-version", kindBinaryStable,
 		fmt.Sprintf("(kind only) This flag can be either %q (build from source) "+
-			"or %q (download a stable binary).", kindBinaryBuild, kindBinaryStable))
+			"or %q (download a stable binary) or a vX.Y.Z version, such as v0.5.1.", kindBinaryBuild, kindBinaryStable))
 	kindClusterName = flag.String("kind-cluster-name", kindClusterNameDefault,
 		"(kind only) Name of the kind cluster.")
+	kindNodeImage = flag.String("kind-node-image", kindNodeImageLatest, "(kind only) name:tag of the node image to start the cluster")
```

Comment:
This should probably default to "" instead, and if we built then it should be replaced with what we built and if not then the kind flag should be unset.

If the flag is actually set, directly pass it through.

Reply:
thanks for your suggestion! I've updated the PR.